commercetools-internal-processors/src/main/java/io/sphere/sdk/annotations/processors/HasUpdateActionsProcessor.java

```diff
@@ -0,0 +1,27 @@
+package io.sphere.sdk.annotations.processors;
+
+import com.squareup.javapoet.JavaFile;
+import io.sphere.sdk.annotations.HasUpdateActions;
+import io.sphere.sdk.annotations.processors.generators.UpdateActionsGenerator;
+
+import javax.annotation.processing.SupportedAnnotationTypes;
+import javax.annotation.processing.SupportedSourceVersion;
+import javax.lang.model.SourceVersion;
+import javax.lang.model.element.TypeElement;
+import java.util.List;
+
```

Comment:
It would be nice if you would add some javadoc here, like we did for the other generators.